crate/cli/src/actions/symmetric/encrypt.rs

```diff
@@ -69,8 +69,9 @@ impl EncryptAction {
             plaintext,
             None,
             self.authentication_data
-                .clone()
+                .as_deref()
                 .map(|s| s.as_bytes().to_vec()),
```

Comment:
Is it KMIP that enforce storing an `Option<String>` in the `EncryptAction`? Because having to clone this is sad. I don't think we need to store it as `String` if KMIP does not need to.